### PR TITLE
fix: twitter to x

### PR DESCRIPTION
**Pull request by gdn992**

### CODE REVIEW

File: src/hooks/useToast.tsx [hunk 1]
```diff
@@ -92,10 +90,10 @@ const useToastWithTweetButton = () => {
     toastWithButton({
       status: "success",
       title,
-      description: "Let others know as well by sharing it on Twitter",
+      description: "Let others know as well by sharing it on X",
       buttonProps: {
         leftIcon: <XLogo weight="fill" />,
-        children: "Tweet",
+        children: "Post",
```
Comment (BrickheadJohnny) [inline src/hooks/useToast.tsx:96]: Share?

File: src/hooks/useToast.tsx [hunk 2]
```diff
@@ -4,24 +4,22 @@ import {
...
+  useToast as chakraUseToast,
+  UseToastOptions,
 } from "@chakra-ui/react"
 import Button from "components/common/Button"
 import XLogo from "static/icons/x.svg"
 import { useRef } from "react"
 
 const useToast = (toastOptions?: UseToastOptions) => {
-  const useToastWithDefaults = chakraUseToast({
+  return chakraUseToast({
```
Comment (BrickheadJohnny) [inline src/hooks/useToast.tsx:16]: We can then remove the `return` and the curly brackets too hehe

File: src/pages/privacy-policy.tsx [hunk 3]
```diff
@@ -383,7 +383,7 @@ const Page = (): JSX.Element => (
           <ListItem>Telegram</ListItem>
           <ListItem>GitHub</ListItem>
           <ListItem>Google</ListItem>
-          <ListItem>Twitter</ListItem>
+          <ListItem>X</ListItem>
```
Comment (BrickheadJohnny) [inline src/pages/privacy-policy.tsx:386]: Let's don't modify the privacy and terms and conditions pages

File: src/components/[guild]/JoinModal/components/TwitterRequirementsVerificationIssuesAlert.tsx [hunk 4]
```diff
@@ -22,7 +22,7 @@ const TwitterRequirementsVerificationIssuesAlert = () => {
       <AlertIcon />
 
       <AlertDescription>
-        Access verification may be limited due to Twitter API issues.
+        Access verification may be limited due to X API issues.
```
Comment (BrickheadJohnny) [inline src/components/[guild]/JoinModal/components/TwitterRequirementsVerificationIssuesAlert.tsx:25]: Please merge back the main branch to this one, I just removed this component, we don't need it anymore

File: src/components/common/Layout/components/NavMenu/NavMenu.tsx [hunk 5]
```diff
@@ -96,7 +96,7 @@ const NavMenu = (): JSX.Element => {
                 rel="noopener"
                 leftIcon={<XLogo />}
               >
-                Twitter
+                X
```
Comment (BrickheadJohnny) [inline src/components/common/Layout/components/NavMenu/NavMenu.tsx:99]: Let's keep it "Twitter" here for now, X (next to another X) looks really strange

**Review by BrickheadJohnny — CHANGES_REQUESTED** [review]

File: src/hooks/useToast.tsx (hunk 2 above)
Comment (gdn992) [reply src/hooks/useToast.tsx:16]: true, I forgot to remove it

File: src/pages/privacy-policy.tsx (hunk 3 above)
Comment (gdn992) [reply src/pages/privacy-policy.tsx:386]: true, it was a mistake, I didn't want to modify this file.

File: src/components/common/Layout/components/NavMenu/NavMenu.tsx (hunk 5 above)
Comment (gdn992) [reply src/components/common/Layout/components/NavMenu/NavMenu.tsx:99]: yeah, looks strange...

Comment (gdn992) [reply src/components/common/Layout/components/NavMenu/NavMenu.tsx:99]: but in this case every place I changed will look strange

**Review by BrickheadJohnny — APPROVED** [review]

**Pull request merged**